ctapipe/tools/dl1_merge.py

```diff
@@ -315,7 +315,8 @@ def add_image_statistics(self, file):
             )
 
             for node in service_nodes:
-                file.copy_node(node, newparent=target_group)
+                if node in file:
```

Comment:
Is this for the case of only dl1b in the file or whats the purpose here?

Reply:
No, this is use for the `__table_column_meta` tables from astropy.